mllib/src/main/scala/org/apache/spark/mllib/stat/impl/MultivariateGaussian.scala

```diff
@@ -54,7 +90,7 @@ private[mllib] class MultivariateGaussian(
    * where k is length of the mean vector.
    * 
    * We here compute distribution-fixed parts 
-   *  (2*pi)^(-k/2) * det(sigma)^(-1/2)
+   *  log((2*pi)^(-k/2) * det(sigma)^(-1/2))
```

Comment:
By the way, in Scala doc (here and elsewhere), I believe the correct way to write exponents is to have an opening ^ and closing ^ around the exponent:
```
det(sigma)^(-1/2)^
```